examples/experimental/u7s.yaml

```diff
@@ -0,0 +1,141 @@
+# Try to use release-yyyyMMdd image if available. Note that release-yyyyMMdd will be removed after several months.
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release-20231010/ubuntu-22.04-server-cloudimg-amd64.img"
+  arch: "x86_64"
+  digest: "sha256:5bed3f233c2422187e86089deea51bb8469dc2a26e96814ca41ff8f14dc80308"
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release-20231010/ubuntu-22.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
+  digest: "sha256:5167c1b13cb33274955e36332ecb7b14f02b71fd19a37a9c1a3a0f8a805ab8e5"
+# Fallback to the latest release image.
+# Hint: run `limactl prune` to invalidate the cache
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-amd64.img"
+  arch: "x86_64"
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
+
+# Mounts are disabled in this template, but can be enabled optionally.
+mounts: []
+# containerd is managed by Docker, not by Lima, so the values are set to false here.
+containerd:
+  system: false
+  user: false
+provision:
+- mode: system
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    loginctl enable-linger ${LIMA_CIDATA_USER}
+- mode: system
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    command -v kubectl >/dev/null 2>&1 && exit 0
+    version=$(curl -L -s https://dl.k8s.io/release/stable.txt)
+    case $(uname -m) in
+      x86_64)   arch=amd64;;
+      aarch64)  arch=arm64;;
+    esac
+    curl -L "https://dl.k8s.io/release/$version/bin/linux/$arch/kubectl" -o /usr/local/bin/kubectl
+    chmod 755 /usr/local/bin/kubectl
+    kubectl version --client
+- mode: system
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    test -d /var/lib/usernetes && exit 0
+    branch=master
+    curl -fsSL https://github.com/rootless-containers/usernetes/archive/$branch.tar.gz | tar -C /var/lib --transform="s/-$branch//" -xz
+    chown ${LIMA_CIDATA_UID} /var/lib/usernetes
+    cd /var/lib/usernetes
+    patch --forward -r - kubeadm-config.yaml <<EOF
+    @@ -7,6 +7,9 @@
+     ---
+     apiVersion: kubeadm.k8s.io/v1beta3
+     kind: ClusterConfiguration
+    +apiServer:
+    +  certSANs:
+    +  - "127.0.0.1"
+     networking:
+       serviceSubnet: "10.96.0.0/16"
+       podSubnet: "10.244.0.0/16"
+    EOF
+- mode: system
+  script: |
+    #!/bin/bash
+    cd /var/lib/usernetes/init-host
+    ./init-host.root.sh
+- mode: user
+  script: |
+    #!/bin/bash
+    cd /var/lib/usernetes/init-host
+    ./init-host.rootless.sh
+- mode: user
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    cd /var/lib/usernetes
+    make up
+    make kubeadm-init
+    # Installing a Pod network add-on
+    make install-flannel
+    # Control plane node isolation
+    kubectl taint nodes --all node-role.kubernetes.io/control-plane-
+    make kubeconfig
+    # Replace the server address with localhost, so that it works also from the host
```

Comment:
Does it still work from the guest too?

Reply:
It is binding to `0.0.0.0` by default

Reply:
So that means "yes", from both.

```console
anders@ubuntu:~$ limactl shell u7s kubectl version
Client Version: v1.28.4
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.28.4
anders@ubuntu:~$ LIMA_INSTANCE=u7s kubectl.lima version
Client Version: v1.28.1
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.28.4

```

Reply:
Same as with the regular k8s